components/DesaDigital/LevelFour/index.vue

```diff
@@ -32,6 +33,7 @@
 </template>
 
 <script>
+import debounce from 'lodash.debounce'
```

Comment:
@yoslie do some of these files have the same template and script?
I think you don't need to write the same file 4 times, you can make this code more modular so it can be reusable. wdyt?

Reply:
I'm about to, but turns out the level four have different display. And this feature still under development on the bussiness side so maybe many changes in the future. If there's no changes and already stable we will refactor this into one single component if possible 🙏